src/server.rs

```diff
@@ -84,7 +84,7 @@ impl Server {
                     return Ok(());
                 }
                 info!(?port, "new client");
-                let listener = match TcpListener::bind(("::", port)).await {
+                let listener = match TcpListener::bind(("0.0.0.0", port)).await {
```

Comment:
Does this work even when `"::"` or `Ipv6Addr::UNSPECIFIED` is passed in? I wanted to make sure that the server was listening on both Ipv4 and Ipv6 in dual-stack mode.

Reply:
It does not with both. That is why I changed it. At least from my testing ipv6 does no work even before that change but can be my network setup

Reply:
Got it, thanks for looking into this and figuring out the issue!